### PR TITLE
feat(@clayui/css): Add new twitter-x icon

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPS-201731

### CODE REVIEW

**Comment by ilzamcmed** [comment]
This PR is ready for review. :tada: 
@pat270 @ethib137

File: packages/clay-css/src/images/icons/twitter-x.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+* SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2022 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
+
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
```
Comment (pat270) [inline packages/clay-css/src/images/icons/twitter-x.svg:8]: We need to replace `width` and `height` with `viewBox="0 0 16 16"`.

File: packages/clay-css/src/images/icons/twitter-x.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <g clip-path="url(#Icon-twitter-x__a)">
```
Comment (pat270) [inline packages/clay-css/src/images/icons/twitter-x.svg:9]: We need to change the indent from spaces to tab.

**Review by pat270 — CHANGES_REQUESTED** [review]

**Comment by pat270** [comment]
One other note, we shouldn't merge this until we finish https://liferay.atlassian.net/browse/LPS-199577. The viewBox="0 0 16 16" will cause https://liferay.atlassian.net/browse/LPS-201063.

**Comment by pat270** [comment]
@ilzamcmed I removed the attribute `fill="#000"` from `packages/clay-css/src/images/icons/twitter-x.svg` and added the license text.

**Review by pat270 — APPROVED** [review]

**Pull request merged**